src/main.py

```diff
@@ -1,16 +1,28 @@
-# This is a sample Python script.
+import unittest
+
+
+class ReturnClass:
+    def return_number(self):
+        return 897
+back = ReturnClass()
+resolution = back.return_number()
+print(f"number = {resolution}")
```

Comment:
This should be fixed during self review. Delete this print and calling of this class